packages/create-expo/src/Examples.ts

```diff
@@ -96,8 +96,15 @@ export async function downloadAndExtractExampleAsync(root: string, name: string)
     throw new Error('Failed to fetch the examples code from https://github.com/expo/examples');
   }
 
+  if (!response.body) {
+    debug(
+      `Failed to fetch the examples code, Github responded without content, received status "${response.status}"`
+    );
+    throw new Error('Failed to fetch the examples code from https://github.com/expo/examples');
+  }
+
   await pipeline(
-    response.body,
+    Readable.from(response.body),
     tar.extract(
```

Comment:
This is a great example of converting web streams to node streams, when necessary.